backend/controllers/uploadController.ts

```diff
@@ -19,7 +19,7 @@ const singleLocal = setLocal.single('image')
 export const uploadLocal = (req: Request, res: Response) => {
   singleLocal(req, res, function (err) {
     if (err) {
-      console.log(err)
+      res.status(400).send(err)
```

Comment:
Rebase from master, this change is already there

Reply:
done